orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -76,6 +81,10 @@ func TestBaseOrchestratorMconfigBuilder_Build(t *testing.T) {
 				LogLevel:       protos.LogLevel_INFO,
 				EventVerbosity: -1,
 			},
+			"state": &mconfig_protos.State{
+				//expected value differs from map due to added jitter
+				SyncInterval: 68,
```

Comment:
Instead of adding a comment to a magic number, make the number a variable with descriptive name. If the comment is still necessary, add it as a doc comment on the variable